docs/NOW-spec.md

```diff
@@ -519,32 +519,6 @@ The NOW_STRING256 structure is used to represent variable-length strings of up t
 
 #### NOW_BUFFER
 
-The NOW_BUFFER structure is used to represent variable-length buffers.
```

Comment:
From my pont of view, there is not much value in this type when we already have more compact `NOW_VARBUF`